app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self, name: str, appetite: int,
+            is_hungry: bool = True) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = not self.is_hungry
```

Comment:
The line `self.is_hungry = not self.is_hungry` toggles the `is_hungry` state every time `feed` is called. This means that if an animal is fed once, it will become not hungry, but if `feed` is called again without checking if the animal is hungry, it will incorrectly toggle back to being hungry. Consider setting `self.is_hungry = False` instead to ensure the animal remains not hungry after being fed.